### PR TITLE
Wiki update

### PR DESCRIPTION
**Pull request by xhanulik**
Second batch of wiki update (after #14).

### CODE REVIEW

**Review by frankmorgner — CHANGES_REQUESTED** [review]

**Review by frankmorgner — CHANGES_REQUESTED** [review]
I looked through the pages and made some general suggestions.

If you find the time, you may also look at the CI's comments regarding the spelling.

Thank you!

File: How-to-write-a-good-bug-report.md [hunk 1]
```diff
@@ -67,7 +69,8 @@ Name the operating system and version you are using, such as "Windows 8.1", or "
 #### File attachments
 
 If you can supplement your bug report with an image or debug log that helps others reproduce the issue, attach these files. You may paste a short (!) log into a pre-formatted block by both, prepending and appending, three backticks to your output:
-````
+
+````text
```
[the remark this answers is not in the document]
Comment (xhanulik) [reply How-to-write-a-good-bug-report.md:73]: This was part of https://github.com/OpenSC/Wiki/commit/588feb014779fe277726693f0c7916339f19ce36, I think it intentional to keep the inner backticks.

File: Resources,-Links.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+[PC/SC Workgroup](https://pcscworkgroup.com/)
+defines the PC/SC standard that is used on windows, but
+thanks to PC/SC-Lite software also on Linux, Mac OS X and BSD.
+You can [download](https://pcscworkgroup.com/specifications/download/)
+all parts of the specification.
+
+Important standards are:
+
+* [PKCS #15](https://www.usenix.org/legacy/publications/library/proceedings/smartcard99/full_papers/nystrom/nystrom_html/index.html): Cryptographic Token Information Format Standard
```
[the remark this answers is not in the document]
Comment (xhanulik) [reply Resources,-Links.md:17]: I think the whitespace should be there as it is written that way also in the linked docs (and the link work fine).

File: Schlumberger-Axalto-Gemalto-e-gate.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# Schlumberger / Axalto / Gemalto e-gate
+
+Schlumberger/Axalto offers the [e-gate adapter](Using-Schlumberger-e-gate-on-Linux), an USB adapter for Schlumberger / Axalto Cryptoflex and Cyberflex cards. See the wiki page of [e-gate](Using-Schlumberger-e-gate-on-Linux) for USB driver information and card pages for card related information.
+
+As of 2009 the e-gate adapter and both Cryptoflex and Cyberflex are discontinued products.
```
[the remark this answers is not in the document]
Comment (xhanulik) [reply Schlumberger-Axalto-Gemalto-e-gate.md:5]: `cyberflex` and `flex` drivers are already _deactivated_ in `old_card_drivers`, so they are on the way to be removed

**Comment by xhanulik** [comment]
Thanks for review, I will fix rest of the spelling errors in the third (and last) PR with updates.

**Pull request merged**